### ISSUE TITLE
Add IntelliSense to Visual Studio for components 

### ISSUE THREAD

**Issue by dotnetnoobie** [microsoft/fast#3454]
**What feature are you requesting?**
Add IntelliSense to Visual Studio for components 

**What solution would be ideal for you?**
IntelliSense for the standard FAST out of the box components and standard html element attributes & events
plus custom/third party components including custom attributes/properties & events

Also for child elements of components for example in **fast-accordion** element Visual Studio would know that it's child elements are **fast-accordion-item** and give a warning if a wrong component was used 

**Is there any additional context for your request?**
Currently there is no IntelliSense in Visual Studio for even standard attributes on components like "class" etc.

**Comment by ThorFjelldalen** [comment]
Is there any new status here? We have a design library written in FAST, but when importing it in other projects, VS Code has no knowledge of the web components in the library. 

After browsing the Discord, I found some mentions of this, using [Custom Elements Manifest](https://custom-elements-manifest.open-wc.org). I tried using  `cem analyze --fast` for generating a `custom-elements.json`, then referencing that file in the `package.json` of the library. I might have done something wrong, but I cannot get that to work. I cannot find any mentions of this in the official FAST docs either.

In the Discord, there were also someone with a working solution in SolidJS, but I'm not sure whether it would work in a project where the custom elements is inside a dependency. https://www.npmjs.com/package/custom-element-solidjs-integration

Anyone gotten intellisense to work for their design library written in FAST, in projects using that design library?
